### PR TITLE
Remove jQuery in Preview.js, use react #1717

### LINKED ISSUE #1717: Remove jQuery in Preview.js, use react
**Actual Behaviour**

The preview for [botbuilder](https://skills.susi.ai/botbuilder/botwizard), jquery is being used for manipulation, which can be removed.

**Expected Behaviour**

Use react methods to eliminate jQuery.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1717 

Changes: 

1. Integrated React properly, using setState to manipulate DOM instead of jQuery. Message to set input message, messages to store array of message. previewChat for toggling
2. Messages set in state, instead of DOM nodes.

**Latest link:** https://pr-1719-3-fossasia-susi-skill-cms.surge.sh

New link after changes: https://pr-1719-2-fossasia-susi-skill-cms.surge.sh

New link after changes: https://pr-1719-1-fossasia-susi-skill-cms.surge.sh

Surge Deployment Link: https://pr-1719-fossasia-susi-skill-cms.surge.sh

### CODE REVIEW

File: src/components/BotBuilder/Preview/Preview.js [hunk 1]
```diff
@@ -24,244 +28,105 @@ class Preview extends Component {
...
     if (this.props.designData !== prevProps.designData) {
-      this.setState(
-        {
-          botbuilderBackgroundBody: this.props.designData
-            .botbuilderBackgroundBody
-            ? this.props.designData.botbuilderBackgroundBody
-            : '#ffffff',
-          botbuilderBodyBackgroundImg: this.props.designData
-            .botbuilderBodyBackgroundImg
+      this.setState({
```
Comment (AakashMallik) [inline src/components/BotBuilder/Preview/Preview.js]: Destructuring this.props may be done here.

File: src/components/BotBuilder/Preview/Preview.js [hunk 2]
```diff
@@ -24,244 +28,105 @@ class Preview extends Component {
...
+    this.setState(prevState => ({
+      previewChat: !prevState.previewChat,
+    }));
   };
 
   // Send request to SUSI API
-  send = text => {
-    let url = urls.API_URL + '/susi/chat.json?q=' + encodeURIComponent(text);
+  sendMessage = event => {
+    let url =
```
Comment (AakashMallik) [inline src/components/BotBuilder/Preview/Preview.js]: template string can be used here.

**Comment by fragm3** [comment]
@hkedia321 Made the changes

**Comment by fragm3** [comment]
@hkedia321 working on point 1, point 2 is actually better this way.

File: src/components/BotBuilder/Preview/Preview.js [hunk 3]
```diff
@@ -24,309 +29,295 @@ class Preview extends Component {
 
   componentDidUpdate(prevProps) {
     if (this.props.designData !== prevProps.designData) {
```
Comment (akshatnitd) [inline src/components/BotBuilder/Preview/Preview.js]: Please destructure the props here.

**Comment by fragm3** [comment]
@akshatnitd made the changes, link: https://pr-1719-3-fossasia-susi-skill-cms.surge.sh

**Comment by fragm3** [comment]
@hkedia321 please review

**Pull request merged**